lib/init.coffee

```diff
@@ -1,8 +1,56 @@
+{CompositeDisposable} = require 'atom'
+
 module.exports =
   config:
-    xmllintExecutablePath:
+    executablePath:
       type: 'string'
-      default: ''
-
+      title: 'xmllint Executable Path'
+      default: 'xmllint'
   activate: ->
     console.log 'activate linter-xmllint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-xmllint.xmllintExecutablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+    console.log 'activate: ' + @executablePath
+    @executablePath = '/usr/bin/xmllint'
+  deactivate: ->
+    @subscriptions.dispose()
+
+  lintFile: (filePath)->
+    helpers = require 'atom-linter'
```

Comment:
I would require this at the start of the file, it may be cached but this is a blocking operation.